Header header
float32 vel_limit_mps
float32 accel_limit_mps2
float32 decel_limit_mps2
float32 dtz_decel_limit_mps2
float32 coastdown_decel_mps2
float32 yaw_rate_limit_rps
float32 yaw_accel_limit_rps2
float32 tire_diameter_m
float32 wheelbase_length_m
float32 wheel_track_width_m
float32 transmission_ratio
uint32 config_bitmap
string eth_ip_address
uint32 eth_port_number
string eth_subnet_mask
string eth_gateway
uint32[] user_feedback_bitmaps
float32 lateral_accel_limit_mps2